src/effects/backends/builtin/balanceeffect.h

```diff
@@ -27,8 +27,13 @@ class BalanceGroupState : public EffectState {
 
 class BalanceEffect : public EffectProcessorImpl<BalanceGroupState> {
   public:
+    // TODO re-evaluate default/delete here. Is adhering to rule of zero possible?
```

Comment:
Not as part of this PR, I just stumbled over this class because the build broke here (transitive header dependency) so this is the best bandaid fix I found.

Reply:
https://github.com/mixxxdj/mixxx/issues/13409